index.html

```diff
@@ -62,5 +62,234 @@ <h2 class="heading_2">
+            <div class="action">
+                <div class="button">
+                    <button class="project" type="submit">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </section>
+    </main>
+    <section class="about_me">
+        <div class="about">
+            <h1 class="primary_text">
+                About me
+            </h1>
+            <p class="description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="btn">
+                <button class="resume" type="submit">
+                    Get My Resume
+                </button>
+            </div>
+        </div>
+        <div class="skill">
+            <div class="Frame _35">
+                <ul class="ul frame _30">
+                    <li>
+                        <img src="images/languages.png" alt="orange 18x18 square rotated -45deg">
+                    </li>
+                    <li class="lang language">
+                        Language
+                    </li>
+                </ul>
+                <ul class="ul Frame _29">
+                    <li class="item">Javascript</li>
+                    <li class="item">Ruby</li>
+                    <li class="item">Html</li>
+                    <li class="item">Css</li>
+                </ul>    
+            </div>
+            <div class="Frame _34">
+                <ul class="ul frame _31">
+                    <li>
+                        <img src="images/frameworks.png" alt="orange 18x18 square">
+                    </li>
+                    <li class="lang frameworks">
+                        Frameworks
+                    </li>
+                </ul>
+                <ul class="ul Frame _29">
+                    <li class="item">Bootstrap</li>
+                    <li class="item">Ruby on Rails</li>
+                    <li class="item">RSPec</li>
+                    <li class="item">CapyBara</li>
+                    <li class="item">Selenium</li>
+                </ul>    
+            </div>
+            <div class="Frame _33">
+                <ul class="ul frame _32">
+                    <li>
+                        <img src="images/skills.png" alt="orange circle">
+                    </li>
+                    <li class="lang skills">
+                        Skills
+                    </li>
+                </ul>
+                <ul class="ul Frame _29 _28">
+                    <li class="item">Codekit</li>
+                    <li class="item">Github</li>
+                    <li class="item">Codepen</li>
+                    <li class="item">Gitlab</li>
+                    <li class="item">Terminal</li>
+                </ul>    
+            </div>
+        </div>
```

Comment:
- [ ] What do you think about using ` <ul class="skill">` instead of a `<div class="skill">`. Then each of  `<div class="Frame _35">` can be replaced with a `<li class="Frame _35">`. This will improve the semantic structure of your website significantly since each of these is infact a `list` of information about you.